File: hackerrank/juneEasyChall/cool.cpp
#include <iostream>
using namespace std;
void binary(int number) {
	int remainder;
	if(number <= 1) {
		cout << number;
		return;
	}

	remainder = number%2;
	binary(number >> 1);    
	cout << remainder;
}

int main(){
	binary(24);
	cout << endl;
	return 0;
}

File: hackerrank/juneEasyChall/cool.py
bin = ""
def binary(num, bin):
	# print bin
	if(num <=1):
		print num
		bin = bin + str(num)
		return
	remainder = num%2
	binary(num >> 1, bin)
	# print remainder
	bin = bin + str(remainder)
	print bin
binary(24, bin)
print "blah " + bin
# bin = ""